compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -772,6 +784,14 @@ public void visit(BLangPackage pkgNode) {
         result = pkgNode;
     }
 
+    // Add global variables with default values to init function
```

Comment:
This method is also self-explanatory right? Do we need the comment?

Reply:
Will remove thanx